prime-router/src/test/kotlin/fhirengine/engine/fhirRouterTests/GetFilterTests.kt

```diff
@@ -85,14 +85,16 @@ class GetFilterTests {
             jurisdictionalFilter = listOf("testOrgJuris"),
             qualityFilter = listOf("testOrgQuality"),
             routingFilter = listOf("testOrgRouting"),
-            processingModeFilter = listOf("testOrgProcMode")
+            processingModeFilter = listOf("testOrgProcMode"),
+            conditionFilter = listOf("testOrgCondition")
         ),
         ReportStreamFilters(
             topic = Topic.COVID_19,
             jurisdictionalFilter = listOf("testCovidJuris"),
             qualityFilter = listOf("testCovidQuality"),
             routingFilter = listOf("testCovidRouting"),
-            processingModeFilter = listOf("testCovidProcMode")
+            processingModeFilter = listOf("testCovidProcMode"),
+            conditionFilter = listOf("testCovidCondition")
```

Comment:
Why was this needed?

Reply:
Ah I see, if you update `ReportStreamFilters` primary constructor in `ReportStreamFilter.kt` to set a default value for `conditionFilter` to be `null`, then you can omit this in this case and in some of the other cases where you're explicitly setting it to null.

Reply:
Good call, thanks for that. Sorry about the bolding. Idk what that was about.